### ISSUE TITLE
Type check error when try to build using ecr version above 0.0.4

### ISSUE THREAD

**Issue by godlike0108** [CircleCI-Public/aws-ecr-orb#47]
### Orb version

Type check error happens when version is above 0.0.4.

### What happened

```
workflows:
  version: 2
  build:
      - aws-ecr/build_and_push_image:
          account-url: ${AWS_ACCOUNT_ID}.dkr.ecr.${AWS_DEFAULT_REGION}.amazonaws.com
          repo: '${AWS_RESOURCE_NAME_PREFIX}'
          region: ${AWS_DEFAULT_REGION}
          tag: '${CIRCLE_SHA1}'
```
This is part of my CircleCI setup.
This works totally fine when version is below 0.0.4

Refer to this [issue#41](https://github.com/CircleCI-Public/aws-ecr-orb/issues/41), I changed orb job name to `aws-ecr/build-and-push-image` when I try to use `circleci/aws-ecr@6.2.0`

However this type check error always showed up when I try to build.
<img width="1515" alt="螢幕快照 2019-08-11 上午9 52 41" src="https://user-images.githubusercontent.com/22214574/62828633-d0859680-bc1d-11e9-9f18-3aa01572a00e.png">

What happened?

### Expected behavior

Build success like v0.0.4.

**Comment by lokst** [comment]
@godlike0108 In newer versions of the orb, some of the parameters are now of the type `env_var_name` instead of `string`. Could you try these changes? (You will need to define a new project env variable called AWS_ACCOUNT_URL or any name that you like) 

```
- aws-ecr/build_and_push_image:
          account-url: AWS_ACCOUNT_URL
          repo:  '${AWS_RESOURCE_NAME_PREFIX}'
          region: AWS_DEFAULT_REGION
          tag: '${CIRCLE_SHA1}'
```

**Comment by lokst** [comment]
@godlike0108 Closing this issue for now, but feel free to reopen it if you still have questions

**Comment by marianobrc** [comment]
I'm having the same issue here. 


I'm trying to build and deploy a docker container to AWS ECR / ECS.

I just started with circle ci, so I went through the steps in this tutorial
https://circleci.com/docs/2.0/ecs-ecr/#section=deployment

I added .circleci/config.yml file to my repo and copied exactly the content of the "Full configuration file" shown here.
https://circleci.com/docs/2.0/ecs-ecr/#full-configuration-file

Then I did set all the env vars in teh project settings, and when i run it, it fails with error 
"Type error for argument region: expected type: env_var_name, actual value: \"${AWS_DEFAULT_REGION}\" (type string)"
![curcleci_env_vars_error](https://user-images.githubusercontent.com/9864478/73696225-9839af00-46ba-11ea-99e9-6a6bac7e5a82.png)


I'm using the same orb versions shown in the tutorial example:
  aws-ecr: circleci/aws-ecr@0.0.10
  aws-ecs: circleci/aws-ecs@0.0.8

I'm wondering if the example in the tutorial is outdated and if you have plans to update it or how could I make it work? 
Thanks
